package.json

```diff
@@ -19,6 +19,9 @@
     "security",
     "zeppelin"
   ],
+  "engines": {
+    "node": ">=8.9.1"
```

Comment:
What are the consequences of adding this? A lot of the Ethereum JS ecosystem is still on previous Node versions.

Reply:
All it does right now is print a warning when someone installs this package as a dependency and isn't running an appropriate version... perhaps it's better to restrict it to `>6`? All we really need is some es6 support, namely const/let/async/await and friends.

Reply:
Gotcha. `async` seems to be available since 7.6.0... We want to support lower versions by compiling anyway (#326), so I'd remove the `"engines"` field to avoid printing the warning for now.

If that sounds good I'll merge after the change!